src/main/scala/firrtl/stage/FirrtlStage.scala

```diff
@@ -0,0 +1,105 @@
+// See LICENSE for license details.
+
+package firrtl.stage
+
+import firrtl._
+
+import firrtl.options.{Stage, Phase, StageUtils, Shell, OptionsException, StageOptions}
```

Comment:
Some un-used imports here

Reply:
Unused imports _below_ this line are cleaned up.